department-of-veterans-affairs/abd-vro
# Chengjie8/issue 2428

## What was the problem?
RabbitMQ user was restricted to only be able to connect via localhost

Associated tickets or Slack threads:
- #2428 

## How does this fix it?[^1]
Create our own rabbitmq.conf file and copy it over in order to override the default settings

## How to test this PR
- deploy the branch in dev environment
- check the k8s pod logs and should no longer see the error mentioned in #2428


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
